src/Microsoft.AspNet.Server.Kestrel/Http/SocketOutput.cs

```diff
@@ -16,11 +16,13 @@ public class SocketOutput : ISocketOutput
     {
         public const int MaxPooledWriteReqs = 1024;
 
-        private const int _maxPendingWrites = 3;
```

Comment:
It made more sense before as the `WriteContext` was more stateful with buffers; whereas now it picks up whatever is available at the time of the write.